tools/bin/check_requirements.sh

```diff
@@ -0,0 +1,68 @@
+printf "Docker ";
+if [[ $(which docker) && $(docker --version) ]]; then
+    printf "is installed"
+  else
+    printf "needs to be installed"
+fi;
```

Comment:
instead of leaving these at github comments, you can add them as comments in the actual script. i think it's worth it because it will help the next person who reads it!

Reply:
actually. i take it back. since you're printing it, it's pretty clear what does what.